README.md

```diff
@@ -473,12 +473,12 @@ Enclosure Group resource for HPE OneView.
 oneview_enclosure_group 'EnclosureGroup_1' do
   client <my_client>
   data <resource_data>
-  logical_interconnect_groups [<LIG_name1>, <LIG_name2>]
+  logical_interconnect_groups ['LIG_name1', { name: 'LIG_name2', enclosureIndex: 1 }]
   action [:create, :create_if_missing, :delete]
 end
 ```
 
-**logical_interconnect_groups:** Array of LIG names used to build the interconnect bay configuration
+**logical_interconnect_groups:** Array of data used to build the interconnect bay configuration. Each item can either be a string containing the LIG name or a hash containing the LIG name and enclosureIndex. Note that the enclosureIndex is not used on API200.
```

Comment:
> Note that the enclosureIndex is not used on API200.

Quick question: It's not used because OneView does not supports or because the SDK does not supports?